doc/classes/Object.xml

```diff
@@ -343,7 +343,7 @@
 				    return "Welcome to Godot 4!"
 
 				func _init():
-				    print(self)       # Prints Welcome to Godot 4!"
+				    print(self)       # Prints Welcome to Godot 4!
```

Comment:
From a quick look, I think for strings we tend to use quotes in the "# Prints ..." comments.
```suggestion
				    print(self)       # Prints "Welcome to Godot 4!"
```

Reply:
This goes against my own rules to [prioritize accuracy for these "Prints" comments](https://github.com/godotengine/godot/pull/99081#issue-2649975342), but that's perhaps worth thinking about another time. I can do the change.

Reply:
To me it seems reasonable to use quotes for `print()`s that are mostly text but omit the quotes when printing other scalar or vector types directly. Especially if you are considering other special cases for vectors as in the linked PR. (Mostly commenting here so you can keep it in mind when reconsidering the guidelines later)

Reply:
Yes, correct, it's a matter of reducing any possible ambiguity.